### PR TITLE
Change all references to the epacems_unitid_epa_plant_crosswalk to ep…

### PR DESCRIPTION
**Pull request by aesharpe**
…acamd_eia

### CODE REVIEW

File: src/pudl_zenodo_storage/zs/metadata.py [hunk 1]
```diff
@@ -74,10 +74,8 @@ def _generate_metadata(data_source_id, data_source_uuid):
 epacems = _generate_metadata("epacems", epacems_uuid)
 
 # Unaltered EPACEMS-EIA Crosswalk archive.
-epacems_unitid_eia_plant_crosswalk_uuid = "40696588-d6ee-11ec-abb9-34363bce6e4c"
-epacems_unitid_eia_plant_crosswalk = _generate_metadata(
-    "epacems_unitid_eia_plant_crosswalk", epacems_unitid_eia_plant_crosswalk_uuid
-)
+epacamd_eia_uuid = "3bad6051-a0c3-43a2-97a6-02f61460ab35"
+epacamd_eia = _generate_metadata("epacamd_eia", epacamd_eia_uuid)
```
Comment (zaneselvans) [inline src/pudl_zenodo_storage/zs/metadata.py:78]: I think this should be the same as the old UUID, not a new UUID -- we aren't creating a new archive lineage are we? It'll just continue to use the existing one, but with a new name.

File: src/pudl_zenodo_storage/frictionless/epacamd_eia.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def datapackager(dfiles):
         https://frictionlessdata.io/specs/data-package/
     """
     return DataPackage.from_resource_archiver(
-        DataSource.from_id("epacems_unitid_eia_plant_crosswalk"),
+        DataSource.from_id("epacamd_eia"),
```
Comment (zaneselvans) [inline src/pudl_zenodo_storage/frictionless/epacamd_eia.py:20]: For this to work you'll need to define the new data source in the PUDL repo, if you haven't already. This repository relies on the `dev` branch of PUDL for its metadata so we need to get those changes merged in there first.

It looks like you'll need to rename the old `epacems_unitid_eia_plant_crosswalk` source that's in `pudl/metadata/sources.py` to the new `epacamd_eia` and update any metadata defined there as appropriate.

**Review by zaneselvans — CHANGES_REQUESTED** [review]

File: src/pudl_zenodo_storage/frictionless/epacamd_eia.py (hunk 2 above)
Comment (aesharpe) [reply src/pudl_zenodo_storage/frictionless/epacamd_eia.py:20]: Hmmm, I already did this and then ran `pip install -e ./` from the pudl repo in the `pudl-zenodo-storage` environment

File: src/pudl_zenodo_storage/zs/metadata.py (hunk 1 above)
Comment (aesharpe) [reply src/pudl_zenodo_storage/zs/metadata.py:78]: I guess technically yes! I'll change it back

Comment (aesharpe) [reply src/pudl_zenodo_storage/zs/metadata.py:78]: I changed it back

**Comment by aesharpe** [comment]
Can you run `zenodo_store epacamd_eia --sandbox --verbose` and see if you're getting this error too? 

```
Traceback (most recent call last):
  File "/Users/austensharpe/mambaforge/envs/pudl-zenodo-storage/bin/zenodo_store", line 33, in <module>
    sys.exit(load_entry_point('pudl-zenodo-storage', 'console_scripts', 'zenodo_store')())
  File "/Users/austensharpe/Desktop/Repos/pudl-zenodo-storage/src/pudl_zenodo_storage/cli.py", line 446, in main
    zenodo_upload_token = os.environ["ZENODO_SANDBOX_TOKEN_UPLOAD"]
  File "/Users/austensharpe/mambaforge/envs/pudl-zenodo-storage/lib/python3.10/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'ZENODO_SANDBOX_TOKEN_UPLOAD'
```

File: src/pudl_zenodo_storage/frictionless/epacamd_eia.py (hunk 2 above)
Comment (zaneselvans) [reply src/pudl_zenodo_storage/frictionless/epacamd_eia.py:20]: Okay this is updated on `dev` and it seems to be working.

**Comment by zaneselvans** [comment]
Nope, no error on my side with the sandbox either.

**Review by zaneselvans — APPROVED** [review]
Seems to be working.  I've been able to create new sandbox and production archives successfully.

**Pull request merged**